po/nb.po

```diff
@@ -9104,6 +8833,7 @@ msgstr ""
 "Kjartan Maraas <kmaraas@gnome.org>, 2000. \n"
 "Tor Harald Thorland <tortho@strigen.com>, 2005-2006.\n"
 "Sigve Indregard <sigve.indregard@gmail.com>, 2006.\n"
+"John Erling Blad <jeblad@gmail.com>, 2020.\n"
```

Comment:
`"John Erling Blad <jeblad@gmail.com>, 2018-2020.\n"` as far as I can see.
BTW I would reorder the list: current users are more interested in the current translator than the history. See "Special strings" in [Adjust the header and special strings](https://wiki.gnucash.org/wiki/Translation#Adjust_the_header_and_special_strings).

Reply:
Some projects are very picky with the ordering! ;)